packages/cli/src/models/compiler/solidity/CompilerProvider.ts

```diff
@@ -205,7 +205,7 @@ async function downloadCompiler(
   localFile: string,
 ): Promise<void> {
   const { version, keccak256: expectedHash, path: versionPath } = build;
-  Loggy.spin(
+  Loggy.onVerbose(
```

Comment:
Why this change? Downloading a compiler is a long-running operation, I think it makes sense to display a spinner for it.